src/input.css

```diff
@@ -0,0 +1,116 @@
+@import url("https://fonts.googleapis.com/css2?family=Montserrat:wght@400;500;600;700;800&display=swap");
+@tailwind base;
+@tailwind components;
+@tailwind utilities;
+
+@layer components {
+  .header {
+    @apply flex justify-between px-[80px] py-[19.75px];
+  }
+  .header__list {
+    @apply space-x-[32px];
+  }
+  .header__link {
+    @apply text-[16px] font-[400] leading-[150%];
+  }
+  .header__button ,.hero__button ,.impressions__button ,.footer__button{
```

Comment:
Instead of selecting classes like this if they have the same style you can use the @apply keyword and apply only one  that is holding the style